.gitignore

```
@@ -36,7 +39,7 @@ cli/test-apps/**
 .DS_Store
 
 # Ignore VSCode settings
-.vscode/
+vscode/
```

```suggestion
.vscode/
```
not sure why this changed? it's almost always `.vscode`

---

also not really sure...